src/pal/prebuilt/idl/clrdata_i.cpp

```diff
@@ -14,15 +14,13 @@
 
 #pragma warning( disable: 4049 )  /* more than 64k source lines */
 
+#include <rpc.h>
```

Comment:
The prebuilt files should not be touched - they should be equal to what MIDL generates on Windows to make sure we don't need to modify them again in the future if the original IDL changes. What was the problem that caused you to make this change?